src/cloud_common/schemas/service_config/aos_config.py

```diff
@@ -35,6 +35,15 @@ class AosConfigSchema(BaseModel):
         ),
     ]
 
+    is_resource_limits: Annotated[
```

Comment:
It is better to name it something like skipResouceLimits, to have default behavior if this var is absent. But as it is named in this way in service config, then let's have the same name here as well.

Reply:
yes, it is the same name as we have in config.yaml
so i think better to have they the same